index.html

```diff
@@ -23,7 +23,7 @@
 		/>
```

Comment:
if `https://*.execute-api.us-west-2.amazonaws.com` is for testing purposes, then I suggest we remove it as we should only include url's here that make it to prod.

Reply:
API can be hosted in multiple regions

Reply:
It is required in pre prod stages and there is no security risk in adding all regions supported by Amazon Location Service